test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -400,6 +400,15 @@
          (is (= (:status response) 200))
          (is (= (:body response) hello-body))))
 
+     ;;(testing "request to jetty successful when a non-traversable directory is created in cert dir"
+     ;;   (let [parent (fs/parent tmp-file)
+     ;;         non-traversable-dir (fs/temp-dir "foo")]
+     ;;     (fs/chmod "-rwx" non-traversable-dir)
+     ;;     (fs/sym-link (fs/file parent "bar") non-traversable-dir)
+     ;;     (let [response (get-request)]
+     ;;       (is (= (:status response) 200))
+     ;;       (is (= (:body response) hello-body)))))
```

Comment:
I could only exact a failure of the watch service with :recursive true by adding a breakpoint in this test and creating a directory owned by `root` in crl directory. Just creating a directory with non-traversable permissions is insufficient. Also this symlink business didn't work. I couldn't think of a way to create a directory owned by root in the test that was sane, so I'm wondering if we want to just say the existing validation (which exercises the crl reload functionality) is sufficient.